leakcanary-android/build.gradle

```diff
@@ -2,6 +2,7 @@ apply plugin: 'com.android.library'
 
 dependencies {
   api project(':leakcanary-analyzer')
+  compile 'com.android.support:support-v4:27.0.1'
```

Comment:
whoa, that's heavy imo.
Can we at least get just the split that has only support fragments?

Reply:
Done